lstchain/tools/lstchain_create_dl2_file.py

```diff
@@ -53,25 +67,19 @@ def __init__(self, **kwargs):
         For getting help run:
         lstchain_create_dl2_file --help
         """
-        self.data_ind = None
-        self.data_src_dep = None
-        self.dl2_ind = None
-        self.dl2_src_dep_level = None
-        self.dl2_src_dep_dict = {}
-        self.reg_energy = None
-        self.reg_disp_vector = None
-        self.cls_gh = None
-
-        self._dict_conf = replace_config(standard_config, self.get_current_config())
 
     def setup(self):
 
-        if self.config_file is None:
-            raise ToolConfigurationError("Please provide configuration file with --config parameter")
+        if not len(self.config["events_filters"]):
+            raise ToolConfigurationError("Information on events filtering not found in config.")
+        if not len(self.config["classification_features"]):
```

Comment:
`self.classification_features`

Reply:
No, that shouldn't be the case.

Reply:
I tried accessing `self.classification_features` on your branch and it works as expected.

Reply:
This is how the config file has to look like:

```json
{
  "ReconstructionHDF5Writer": {
    "regression_features": ["intensity", "width", "length"],
    "classification_features": ["intensity", "width", "length"]
  }
}

```

Reply:
Maybe you were missing the `ReconstructionHDF5Writer` level?

Reply:
I have an empty list in `self.classification_features` and the list below in `self.config["classification_features"]`

```
['log_intensity', 'width', 'length', 'x', 'y', 'psi', 'phi', 'wl', 'skewness', 'kurtosis', 'r', 
'time_gradient', 'leakage_intensity_width_2', 'log_reco_energy', 'reco_disp_dx', 'reco_disp_dy']
```

Reply:
Then  you are missing the `ReconstructionHDF5Writer` level in the file. You need to add that.

Reply:
Unfortunately, the config system doesn't raise errors for unrecognised stuff in the config file. Otherwise you would have gotten that.

Reply:
Would you suggest to wrap the existing `regression_features` and `classification_features` info with `ReconstructionHDF5Writer`? In that case I guess all other scripts grabbing that info would need to be modified..

Reply:
This is how the configuration system works! Traitlets have to be in the namespace of the tool in their config file. Like the example I showed above.

Reply:
Ok, just to be sure :) 

I will modify the config files as well as all other scripts using classification/regression features and events filtering accordingly, so they can access the info from the config files.

Reply:
This will also bring us away from the one *monster* lstchain config file to config files per tool, which is I think a good thing.